tests/rtp-mode.rs

```diff
@@ -0,0 +1,140 @@
+    let rtc2 = Rtc::builder()
+        .set_rtp_mode(true)
+        // release packet straight away
+        .set_reordering_size_audio(0)
+        .build();
+
+    let mut l = TestRtc::new_with_rtc(info_span!("L"), rtc1);
+    let mut r = TestRtc::new_with_rtc(info_span!("R"), rtc2);
+
+    let host1 = Candidate::host((Ipv4Addr::new(1, 1, 1, 1), 1000).into())?;
+    let host2 = Candidate::host((Ipv4Addr::new(2, 2, 2, 2), 2000).into())?;
+    l.add_local_candidate(host1);
+    r.add_local_candidate(host2);
+
+    let mut change = l.sdp_api();
+    let mid = change.add_media(MediaKind::Audio, Direction::SendRecv, None, None);
+    let (offer, pending) = change.apply().unwrap();
+
+    let answer = r.rtc.sdp_api().accept_offer(offer)?;
+    l.rtc.sdp_api().accept_answer(pending, answer)?;
+
+    loop {
+        if l.is_connected() || r.is_connected() {
+            break;
+        }
+        progress(&mut l, &mut r)?;
+    }
+
+    let max = l.last.max(r.last);
+    l.last = max;
+    r.last = max;
+
+    let params = l.media(mid).unwrap().payload_params()[0];
+    assert_eq!(params.spec().codec, Codec::Opus);
+    let pt = params.pt();
+
+    let mut exts = ExtensionMap::empty();
+    exts.set(3, Extension::AudioLevel);
+
+    let to_write = vec![
+        // 1
+        &[
+            //
+            144, 33, 183, 152, 0, 0, 39, 16, 0, 0, 0, 44, 190, 222, 0, 1, 48, 170, 0, 0,
+            // payload
+            0x1, 0x2, 0x3, 0x4,
+        ],
+        // 3
+        &[
+            //
+            144, 33, 183, 155, 0, 0, 54, 176, 0, 0, 0, 44, 190, 222, 0, 1, 48, 172, 0, 0,
+            // payload
+            0x9, 0xa, 0xb, 0xc,
+        ],
+        // 2
+        &[
+            //
+            144, 161, 183, 153, 0, 0, 46, 224, 0, 0, 0, 44, 190, 222, 0, 1, 48, 171, 0, 0,
+            // payload
+            0x5, 0x6, 0x7, 0x8,
+        ],
+    ];
+
+    let mut to_write: VecDeque<_> = to_write.into();
+
+    loop {
+        if let Some(packet) = to_write.pop_front() {
+            let wallclock = l.start + l.duration();
+
+            l.media(mid)
+                .unwrap()
+                .writer(pt)
+                .write_rtp(wallclock, packet, &exts)?;
+        }
+
+        progress(&mut l, &mut r)?;
+
+        if l.duration() > Duration::from_secs(10) {
+            break;
+        }
+    }
+
+    let media: Vec<_> = r
+        .events
+        .iter()
+        .filter_map(|e| {
+            if let Event::MediaData(v) = e {
+                Some(v)
+            } else {
+                None
+            }
+        })
+        .collect();
+
+    assert_eq!(media.len(), 3);
+
+    // no change from standard above
+    let exts = ExtensionMap::standard();
+
+    let h0 = RtpHeader::parse(&media[0].data, &exts).unwrap();
+    let h1 = RtpHeader::parse(&media[1].data, &exts).unwrap();
+    let h2 = RtpHeader::parse(&media[2].data, &exts).unwrap();
+
+    assert_eq!(h0.sequence_number, 47000);
```

Comment:
maybe add negative tests for things that should be different for each leg, ssrc etc?